package.json

```diff
@@ -13,23 +13,19 @@
     "heroku-postbuild": "gulp build",
     "lint:prettier": "prettier --config --write '**/*.js'",
     "test": "DEV=true mocha -r ts-node/register './test/**/*.ts' --exit",
-    "start:backend": "UPDATE_DATA=true DEV=true ts-node backend/app.ts"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
+    "start:backend": "UPDATE_DATA=true DEV=true ts-node backend/app.ts",
+    "prepare": "husky install",
+    "pre-commit": "lint-staged"
   },
   "lint-staged": {
     "**/*.js": [
-      "prettier --write",
-      "git add"
-    ]
+      "prettier --write"
+    ],
+    "./backend/*.ts": "eslint --cache"
```

Comment:
only adding to backend for now. During frontend refactor we can include it, the eslint rules may need altering.